src/pages/CircleAdminPage/CircleAdminPage.tsx

```diff
@@ -323,13 +346,48 @@ export const CircleAdminPage = () => {
     }
   };
 
+  const getDecimals = ({ symbol }: { symbol: string | undefined }) => {
```

Comment:
this has been updated in DistributionForm to take in `vault_id` instead

Reply:
done